internal/transform/scope-html_test.go

```diff
@@ -0,0 +1,123 @@
+package transform
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+	"testing"
+
+	astro "github.com/snowpackjs/astro/internal"
+	tycho "github.com/snowpackjs/astro/internal"
+	"golang.org/x/net/html/atom"
+)
+
+func printToSource(buf bytes.Buffer, node *tycho.Node) string {
```

Comment:
Added a specific printer for this test, can extract to a shared util if it continues to be useful.

Reply:
Nice! Yeah printer test probably needs to be broken out at some point